cockatrice/src/tab.h

```diff
@@ -33,6 +33,7 @@ protected slots:
     virtual void retranslateUi() = 0;
     virtual void closeRequest() { }
     virtual void tabActivated() { }
+    virtual bool updateIcon() { return true; }
```

Comment:
It is I believe technically correct - you aren't using the *virtual* method implementation but rather the concrete one.It is I believe technically correct - you aren't using the *virtual* method implementation but rather the concrete one.